homa228.py

```diff
@@ -0,0 +1,16 @@
+print ('ax**2+bx+c=0')
+a = int(input())
```

Comment:
Add print message for each variables

Reply:
in english?

Reply:
It would be good if you add validation for each variable. When I type string, I see exception message